docs/standard/garbage-collection/fundamentals.md

```diff
@@ -267,7 +267,7 @@ Concurrent garbage collection
 
 ## Background workstation garbage collection
 
-In background garbage collection, ephemeral generations (0 and 1) are collected as needed while the collection of generation 2 is in progress. There is no setting for background garbage collection; it is automatically enabled with concurrent garbage collection. Background garbage collection is a replacement for concurrent garbage collection. As with concurrent garbage collection, background garbage collection is performed on a dedicated thread and is applicable only to generation 2 collections.
+In background garbage collection, ephemeral generations (0 and 1) are collected as needed while the collection of generation 2 is in progress. There is no setting for background garbage collection; it is automatically enabled with the setting for concurrent garbage collection. Background garbage collection is a replacement for concurrent garbage collection in the .NET Framework 4 and later versions. As with concurrent garbage collection, background garbage collection is performed on a dedicated thread and is applicable only to generation 2 collections.
```

Comment:
```suggestion
Background garbage collection replaces concurrent workstation garbage collection starting with the .NET Framework 4 and concurrent server garbage collection starting with the .NET Framework 4.5.  In background garbage collection, ephemeral generations (0 and 1) are collected as needed while the collection of generation 2 is in progress. It is performed on a dedicated thread and is applicable only to generation 2 collections. Background garbage collection is automatically enabled by default, and can be enabled or disabled with the [\<gcConcurrent>](../../../docs/framework/configure-apps/file-schema/runtime/gcconcurrent-element.md) configuration setting in .NET Framework applications. 
```

Reply:
ahh, I thought because this is only for the workstation version which was why it didn't mention server GC. 
this 

"Background garbage collection replaces concurrent workstation garbage collection starting with the .NET Framework 4 and concurrent server garbage collection starting with the .NET Framework 4.5. "

is great to mention. do you think it should be mentioned in workstation GC section or have a separate paragraph just to say this? I don't have a strong opinion.

Reply:
Note that there's a Note box directly below this paragraph that explains workstation vs. server.